build_tools/travis/test_script.sh

```diff
@@ -8,6 +8,7 @@
 
 set -e
 
+
```

Comment:
This was added by mistake during the days when travis went down, when I foolishly tried to make travis work in order to get my pr pass tests as this was my first one.Will correct it.